sdk/servicebus/azure_messaging_servicebus/README.md

```diff
@@ -0,0 +1,185 @@
+<!-- cspell:words pwsh yourgroup westus servicebus checkpointing  -->
+
+# Azure Service Bus client library for Rust
+
+Azure Service Bus is a fully managed enterprise message broker with message queues and publish-subscribe topics. Service Bus is used to decouple applications and services from each other, providing the following benefits:
+
+-   Load-balancing work across competing workers
+-   Safely routing and transferring data and control across service and application boundaries
+-   Coordinating transactional work that requires a high-degree of reliability
+
+[Source code] | [Package (crates.io)] | [API reference documentation] | [Product documentation]
+
+## WARNING - Not production ready crate
+
+This crate is in early development, it **SHOULD NOT** be used in production.
+
+## Getting started
+
+### Prerequisites
+
+-   Rust 1.85.0 or later
+-   An Azure subscription
+-   A Service Bus namespace
+
+### Install dependencies
+
+Add the following crates to your project:
+
+```sh
+cargo add azure_identity tokio
```

Comment:
Nit: why `tokio`? EH only did this because @LarryOsterman uses and displays `#[tokio::main]`, which you don't...which I actually prefer because it more closely aligns with how async doc comment examples are supposed to be done (and I recently fixed in #2838 as I was fixing some related things).